README.rst

```diff
@@ -39,6 +39,16 @@ License
 ``giotto-ph`` is distributed under the AGPLv3 `license <https://github.com/giotto-ai/giotto-tda/blob/master/LICENSE>`_.
 If you need a different distribution license, please contact the `L2F team`_.
 
+Parallel persistent homology backend
+====================================
+
+Computing persistence barcodes of large datasets and in high homology degrees is challenging even on modern hardware. ``giotto-ph``'s persistent homology backend
+is able to distribute the key stages of the computation (namely, search for apparent pairs and coboundary matrix reduction) across an arbitrary number of available CPU threads.
+The scaling is generally quite favourable as shown in the following figure (for more details, see our paper linked below):
+
+.. image:: https://raw.githubusercontent.com/giotto-ai/giotto-ph/readme_speedup_figure/docs/images/multithreading_speedup.svg
```

Comment:
```suggestion
.. image:: https://raw.githubusercontent.com/giotto-ai/giotto-ph/main/docs/images/multithreading_speedup.svg
```
To be changed after review

Reply:
@MonkeyBreaker FYI, the reason for not using relative paths is that, when we were doing this in ``giotto-tda``, the image was not rendered in the PyPI readme

Reply:
Oh ! I was not aware, good to know !

Reply:
Done. Ready as far as I am concerned. Let me know :)

Reply:
You can merge